cmd/queue/main_test.go

```diff
@@ -149,7 +149,7 @@ func TestQueueTraceSpans(t *testing.T) {
 				h := queue.ProxyHandler(breaker, network.NewRequestStats(time.Now()), true /*tracingEnabled*/, proxy)
 				h(writer, req)
 			} else {
-				h := health.ProbeHandler(healthState, tc.prober, true /* isAggresive*/, true /*tracingEnabled*/, nil)
+				h := health.ProbeHandler(healthState, tc.prober, true /* isAggressive*/, true /*tracingEnabled*/, nil)
```

Comment:
👍

Reply:
Oh, I should point out that the first commented item has a leading space (but no trailing), whereas the second has no whitespace. I suspect the latter is the preferred style, but I'm not eager to make changes :-)